### ISSUE TITLE
Race issues (Donkey town Eves block new Eve spawns?)

### ISSUE THREAD

**Issue by laccy** [jasonrohrer/OneLifeData7#754]
Today I played several lives

The brown eve was in donkey town, and each family was stuck at Newcomen tech for hours, and most families even died out.
I checked it at age 7 on a waystone, it was 20000 20000 away.
I checked it at 59 and it was still in donkey town.

Next life a new family popped up but they had no females by the time I was old.

Donkey town should not matter, should not take away the specialists from the middle towns.

Also, the brown family had 5 males after the females died. Not everyone realizes this or continues to play. I asked the guy to give rubber and palm, by using a note, he was afk in the fire. He dropped his knife so I knew he was there sometimes. I aimed at him, I did not want to kill him just hoped the sound would make him react. He took my horse cart and dumped in the jungle then I followed with another, he took that one too and dumped into another jungle. I was ginger so I could not take it.
He went back to fire and AFK some more. I choose to kill him since he did not cooperate and I figured that no brown eve will spawn until he is alive.
After he died a brown family popped up.

Males should not matter if there is no female, the next Eve should spawn if there are no females of that race. Donkey town should not matter, people need the rubber, we barely have 45-50 people, 1 of each family should be always present (whites not needed so they should be last when all 3 other race is present). Meanwhile, we had 2 ginger families and 1-2 whites, and almost 2 hours or more without any brown person close to the cities. Also, the new eve camp died out before they had an upgraded well, they only had 1 single bucket and they did not leave any palm or rubber behind. I even tried SID several times to become brown and I could not. 20-25 people out of 50 were gingers. They were overwhelmed by babies and out of water.

**Comment by jasonrohrer** [comment]
I'm pretty sure it already works this way.

It counts fertile females and girls of a race, ignoring males, and ignoring tutorial, d-town, etc players.

Perhaps players rang the bell in d-town and attracted another non-dtown family there?

**Comment by laccy** [comment]
Since I can track it on hetuw, I had data of families popping up and die out, the number of females, total number, even went to the place where the last brown guy lived. When I killed him a new Eve popped up in 2 minutes, not until he was alive, happened again today, I was there when the Rain family died out, while the males were alive, no new Eve spawned. When the last guy died, a new family popped up, not sure of the family name. A white eve was waiting to take over their well, it wasn't possible.

Also, some sort of weird thing going on with the wells, not sure if it's intentional or not. The town with the oil, Old Lebrasseur camp had a spring 40 tiles away. We were using cistern water mostly, we were able to make a new well 40 tiles next to us. That ginger family had 3 outposts, ours died out with 6 males no female bb. Later when I got back there with the Mcnamaras there was no well there, the other one said "abandoned engine well" but we weren't able o claim it, even that we had no other well, and we used that one multiple times.
I saw similar things a few times. A black family had 2 wells 80 distance, when I went back there, I could use the left side one to claim the homeland and the other was usable still. Another family had a secondary well 40 away. Also the new McNamara camp had a spot for a well 80-80 away NE.

Something is wrong with the homelands, they don't tap out correctly unless is intentional or side effect of the changes. Not sure if that's the reason eves don't spawn correctly. Do males still own a homeland and that's why the eve doesn't spawn? I was able to claim new homelands as male so that side is clear. The "doomed" families with no females are still holding onto the homeland that's also sure. 

The d-town player was solo 1-1 means it's a fertile female eve who did not get kids yet(generally tutorial or donkey eve, generally named Jones if tutorial), normal eves fluctuate with 1-3 females, and it isn't an easy task to attract anyone there, it's 20000 tiles, there was no bell this time as it was after the wipe. I saw the coordinates on the expert waystone and Hetuw saves it separately. There were multiple times where only 3 families exist and one of them is overpopulated and still, no eve spawns. Especially brown Eves are missing. White Eves seem to have priority over them. We got male majority outposts with 20-25 people out of 50 while no 4th race or at least not a stable one.

If I play I make a few screenshots about how the population changes and I will try to track down the 4th race.
